engines/composer/detection_tables.h

```diff
@@ -148,7 +148,7 @@ static const ComposerGameDescription gameDescriptions[] = {
 			"imoking",
 			0,
 			{
-				{"book.mac", GAME_CONFIGFILE, "4896a22874bb660f5ba26a0af111f9c0", 1868},
+				{"book.mac", GAME_CONFIGFILE, "916757dbb058581cff857b055ec97642", 2560},
```

Comment:
Please use ADGF_MACRESFORK to make hashes independent from extraction method

Reply:
Awesome, thank you